### PR TITLE
Some correction to 3rdParty algorithms (R-Stat)

### PR DESCRIPTION
**Pull request by ghtmtt**

### CODE REVIEW

File: source/docs/user_manual/processing/3rdParty.rst [hunk 1]
```diff
@@ -210,7 +210,8 @@ entry in the processing configuration dialog. Once you have set that parameter,
 you can start creating and executing your own R scripts.
 
 .. note:: for **Windows** user, usually the R executable file is in the
-   ``C:\Program Files\R\R-3.2``, be aware not to use ``C:\Program Files\R\R-3.2\bin``!
+   ``C:\Program Files\R\R-3.2`` folder.  Add just the folder and **NOT** the the
+   binary!
```
Comment (yjacolin) [inline source/docs/user_manual/processing/3rdParty.rst]: double `the the`

File: source/docs/user_manual/processing/3rdParty.rst [hunk 2]
```diff
@@ -346,16 +347,14 @@ own scripts.
...
-   add the necessary commands at the beginning of your script. You also have to
-   make sure that the corresponding packages are installed in the R distribution
-   used by QGIS. The processing framework will not take care of any package installation. If you
-   run a script that requires a package that is not installed, the execution will fail, and
-   Processing will try to detect which packages are missing. You must install those
-   missing libraries manually before you can run the algorithm.
+   additional libraries that you might need have to be explicitly loaded by typing,
+   ``library(ggplot2)``. If the package is not already installed on your machine,
+   Processing will download and install it. In this way the package will be also
+   available in R Standalone. **Be aware** that if the package has so be downloaded,
```
Comment (DelazJ) [inline source/docs/user_manual/processing/3rdParty.rst]: so --> to

File: source/docs/user_manual/processing/3rdParty.rst [hunk 3]
```diff
@@ -301,11 +302,11 @@ explicitly into a ``SpatialPointsDataFrame`` object, since it is itself an objec
 of class ``ppp``, which is not a suitable class to be returned to QGIS.
 
 If your algorithm generates raster layers, the way they are saved will depend on
-whether or not you have used the ``#dontuserasterpackage`` option. In you have
+whether or not you have used the ``##dontuserasterpackage`` option. In you have
```
Comment (DelazJ) [inline source/docs/user_manual/processing/3rdParty.rst]: In --> If

File: source/docs/user_manual/processing/3rdParty.rst (hunk 2 above)
Comment (yjacolin) [inline source/docs/user_manual/processing/3rdParty.rst]: has **to** be downloaded?

File: source/docs/user_manual/processing/3rdParty.rst [hunk 4]
```diff
@@ -210,7 +210,8 @@ entry in the processing configuration dialog. Once you have set that parameter,
 you can start creating and executing your own R scripts.
 
 .. note:: for **Windows** user, usually the R executable file is in the
-   ``C:\Program Files\R\R-3.2``, be aware not to use ``C:\Program Files\R\R-3.2\bin``!
+   ``C:\Program Files\R\R-3.2`` folder.  Add just the folder and **NOT** the the
```
Comment (yjacolin) [inline source/docs/user_manual/processing/3rdParty.rst]: we can use :file:`C:\Program Files\R\R-3.2` instead of ``C:\Program Files\R\R-3.2``

Comment (DelazJ) [inline source/docs/user_manual/processing/3rdParty.rst]: the ~~the~~

Comment (DelazJ) [inline source/docs/user_manual/processing/3rdParty.rst]: Wasn't it more explicite to cite the folder not to add?

Comment (ghtmtt) [inline source/docs/user_manual/processing/3rdParty.rst]: it was explicit on the binary file not to add yes.. But, I noticed that people read just the last words, and they always do the wrong thing. If you agree we can let just the right path to add

Comment (DelazJ) [inline source/docs/user_manual/processing/3rdParty.rst]: Indeed, if people do not take care of what they read.... better do not give them opportunity to fail. Let's keep your changes...

**Comment by DelazJ** [comment]
LGTM.

File: source/docs/user_manual/processing/3rdParty.rst [hunk 5]
```diff
@@ -346,16 +347,14 @@ own scripts.
...
-   make sure that the corresponding packages are installed in the R distribution
-   used by QGIS. The processing framework will not take care of any package installation. If you
-   run a script that requires a package that is not installed, the execution will fail, and
-   Processing will try to detect which packages are missing. You must install those
-   missing libraries manually before you can run the algorithm.
+   additional libraries that you might need have to be explicitly loaded by typing,
+   ``library(ggplot2)``. If the package is not already installed on your machine,
+   Processing will download and install it. In this way the package will be also
+   available in R Standalone. **Be aware** that if the package has to be downloaded,
+   the first time you run the script it might take a long time.
```
Comment (DelazJ) [inline source/docs/user_manual/processing/3rdParty.rst]: Does this description also apply to 2.14? in which case we can backport this fix.

**Comment by ghtmtt** [comment]
I don't have (right now) the chance to test it on 2.14, but AFAIK the code did not changed from 2.14 to 2.16.

**Comment by DelazJ** [comment]
Ok, When you can check....
Otherwise, if you are ok with this PR, it can be merged

**Comment by ghtmtt** [comment]
Some other checks (especially in Windows) could be very useful. Anyway, I made some other tests on my Linux and it works as described. We can wait until I'm able to run some test on Virtual Box and using Windows

**Comment by DelazJ** [comment]
no hurry imo. It has been there for months, years(?) so yes, we can wait :)

**Comment by ghtmtt** [comment]
Exactly ;)

**Comment by ghtmtt** [comment]
Tested also on Windows and it works. Even for 2.14. You can merge it

**Comment by DelazJ** [comment]
Thanks

**Pull request merged**